hc/front/tests/test_my_checks.py

```diff
@@ -66,11 +66,18 @@ def test_it_shows_purple_check(self):
         self.client.get("/ping/%s/" % self.check.code)
         self.check.refresh_from_db()
 
+    def test_it_shows_nag_check(self):
+        self.client.login(username="alice@example.org", password="password")
+
+        self.check.last_ping = timezone.now() - td(days=3)
+        self.check.status = "nag"
+        self.check.save()
+
```

Comment:
Great job adding tests for better coverage.

Reply:
Good work on writing  tests